setup.py

```diff
@@ -4,15 +4,20 @@
 #
 # This source code is licensed under the AGPLv3 license found in the
 # LICENSE file in the root directory of this source tree.
+import distutils.command.build
```

Comment:
The problem was not that I couldn't figure out a name for that module. ;) (Even though naming is hard.)